Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,32 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            String name;
+            int age;
+            double height;
+            double weight;
+            String cont = "Y";
+            while (cont.Equals("Y") || cont.Equals("y"))
+            {
+                Console.Write("What is your name?");
+                name = Console.ReadLine();
+                Console.Write("How old are you?");
+                age = int.Parse(Console.ReadLine());
+                Console.WriteLine("How tall are you? In cm ");
+                if (!double.TryParse(Console.ReadLine(), out height))
+                {
+                    Console.WriteLine("Something is wrong with your height");
```

Comment:
After a problem with parsing provided height you are continuing a program. That's a flaw in the logic because at the end of a program you will provide incorrect results for a user.

Never continue program if you lack a data to provide correct result, it's better to fail early, escallate that problem has occured and there is no point to continue. It's better to show no result at all than showing incorrect result.